go/engine/list_tracking.go

```diff
@@ -59,33 +59,34 @@ func (e *ListTrackingEngine) RequiredUIs() []libkb.UIKind { return []libkb.UIKin
 func (e *ListTrackingEngine) SubConsumers() []libkb.UIConsumer { return nil }
 
 func (e *ListTrackingEngine) Run(ctx *Context) (err error) {
-	var user *libkb.User
+
+	var arg libkb.LoadUserArg
+
 	if len(e.arg.ForAssertion) > 0 {
-		user, err = libkb.LoadUser(libkb.NewLoadUserByNameArg(e.G(), e.arg.ForAssertion))
+		arg = libkb.NewLoadUserByNameArg(e.G(), e.arg.ForAssertion)
 	} else {
-		user, err = libkb.LoadMe(libkb.NewLoadUserArg(e.G()))
-	}
-	if err != nil {
-		return
+		arg.Self = true
 	}
 
-	var trackList TrackList
-	trackList = user.IDTable().GetTrackList()
+	err = e.G().FullSelfCacher.WithUser(arg, func(user *libkb.User) error {
```

Comment:
I think I get it....this won't cache if not self, and won't use self if not self.